cli/components/input.go

```diff
@@ -79,30 +82,40 @@ func (m password) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	return m, cmd
 }
 
-func (m password) View() string {
+func (m inputModel) View() string {
 	return m.textInput.View() + "\n"
 }
 
-// AskPassword will only prompt the user for a password if they are on a TTY.
-func AskPassword(prompt string) (string, error) {
-	// check if password is set
+// AskInput will only prompt the user for input if they are on a TTY.
+func askInput(prompt string, password bool) (string, error) {
```

Comment:
non-functional code shuffling. this now allows us to create models where we dont ask for paswords via `AskInput`. `AskPassword` remains unchanged.